distribution/test/itests/test-itests-alliance/src/test/java/alliance/test/itests/TestFederation.java

```diff
@@ -0,0 +1,219 @@
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package alliance.test.itests;
+
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.containsString;
+import static org.junit.Assert.fail;
+import static org.ops4j.pax.exam.CoreOptions.maven;
+import static org.ops4j.pax.exam.CoreOptions.mavenBundle;
+import static org.ops4j.pax.exam.CoreOptions.options;
+import static org.ops4j.pax.exam.CoreOptions.wrappedBundle;
+import static org.ops4j.pax.exam.karaf.options.KarafDistributionOption.karafDistributionConfiguration;
+import static org.ops4j.pax.exam.karaf.options.KarafDistributionOption.keepRuntimeFolder;
+
+import static com.jayway.restassured.RestAssured.given;
+import static com.jayway.restassured.RestAssured.when;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.Dictionary;
+import java.util.HashMap;
+import java.util.Hashtable;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.ops4j.pax.exam.Option;
+import org.ops4j.pax.exam.junit.PaxExam;
+import org.ops4j.pax.exam.spi.reactors.ExamReactorStrategy;
+import org.ops4j.pax.exam.spi.reactors.PerClass;
+import org.osgi.service.cm.Configuration;
+
+import com.jayway.restassured.path.xml.XmlPath;
+import com.jayway.restassured.response.ValidatableResponse;
+
+import ddf.common.test.AfterExam;
+import ddf.common.test.BeforeExam;
+import ddf.test.itests.AbstractIntegrationTest;
+
+
+/**
+ * Tests the Alliance additions to DDF framework components.
+ */
+@RunWith(PaxExam.class)
+@ExamReactorStrategy(PerClass.class)
+public class TestFederation extends AbstractIntegrationTest {
+
+    private Thread mockServerThread;
+
+    private static final String[] REQUIRED_APPS = {"catalog-app", "solr-app", "spatial-app",
+        "nsili-app"};
+
+    private static final String NSILI_SOURCE_ID = "nsiliSource";
+
+    @Override
+    protected Option[] configureDistribution() {
+        return options(
+                karafDistributionConfiguration(
+                        maven().groupId("org.codice.alliance.distribution").artifactId("alliance")
+                                .type("zip").versionAsInProject().getURL(),
+                        "alliance", KARAF_VERSION).unpackDirectory(new File("target/exam"))
+                                .useDeployFolder(false));
+    }
+
+    @Override
+    protected Option[] configureCustom() {
+        return options(
+                wrappedBundle(mavenBundle("ddf.test.itests", "test-itests-ddf").classifier("tests")
+                        .versionAsInProject()).bundleSymbolicName("test-itests-ddf")
+                                .exports("ddf.test.itests.*"),
+                wrappedBundle(mavenBundle().groupId("org.codice.alliance.distribution")
+                        .artifactId("sample-nsili-server").versionAsInProject()),
+                wrappedBundle(mavenBundle().groupId("ddf.catalog.core")
```

Comment:
Wrapping bundles can be dangerous and avoided.  This will expose the classes within the jar and could potentially mask a deployment issue. I think we need to remove these and get the tests running with only the ddf-itests & sample server jar.